.github/workflows/release-brew.yaml

```diff
@@ -3,21 +3,118 @@ on:
   release:
     types: [created]
 
+env:
+  FORMULA: cbmc-viewer
+  TAP: aws/tap
+  BOT_USER: aws-viewer-for-cbmc-release-ci
+  RELEASE_TAG: ${GITHUB_REF/refs\/tags\/}
+  VERSION: $(echo ${{ github.ref }} | cut -d "/" -f 3 | cut -d "-" -f 2)
+  FORK_REPO: https://${{ secrets.RELEASE_CI_ACCESS_TOKEN }}@github.com/$BOT_USER/homebrew-$(echo $TAP |cut -d "/" -f 2).git
+
 jobs:
   homebrew-pr:
     name: Homebrew Bump Formula PR
     runs-on: macos-10.15
     steps:
-      - name: Get release tag name
-        run: echo "RELEASE_TAG=${GITHUB_REF/refs\/tags\/}" >> $GITHUB_ENV
       - name: Configure git user name and email
         uses: Homebrew/actions/git-user-config@master
         with:
-          username: aws-viewer-for-cbmc-release-ci
+          username: ${{ env.BOT_USER }}
+
       - name: Create homebrew PR
         run: |
+          brew tap ${{ env.TAP }}
           brew update-reset
-          brew tap aws/tap
-          brew bump-formula-pr --tag "$RELEASE_TAG" --revision "$GITHUB_SHA" aws/tap/cbmc-viewer
+          brew bump-formula-pr --tag "${{ env.RELEASE_TAG }}" --revision "$GITHUB_SHA" ${{ env.TAP }}/${{ env.FORMULA }}
         env:
           HOMEBREW_GITHUB_API_TOKEN: ${{ secrets.RELEASE_CI_ACCESS_TOKEN }}
+
+  build-bottle:
+    needs: homebrew-pr
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, macos-10.15]
+    runs-on: ${{ matrix.os }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Checkout PR
+        run: |
+          brew tap ${{ env.TAP }}
+          brew update-reset
+          cd $(brew --repo ${{ env.TAP }})
+          git remote add fork-repo ${{ env.FORK_REPO }}
+          git fetch fork-repo
+          git checkout -B bump-${{ env.FORMULA }}-${{ env.VERSION }} fork-repo/bump-${{ env.FORMULA }}-${{ env.VERSION }}
+
+      - name: Tap Syntax
+        run: |
+          brew audit --online --git --skip-style ${{ env.TAP }}/${{ env.FORMULA }}
+          brew style ${{ env.TAP }}/${{ env.FORMULA }}
+
+      - name: Build bottle
+        run: |
+          brew test-bot --tap ${{ env.TAP }} --testing-formulae ${{ env.TAP }}/${{ env.FORMULA }} --only-formulae --root-url=https://github.com/$GITHUB_REPOSITORY/releases/download/${{ env.RELEASE_TAG }}
+
+      - name: Get Package Path
+        id: get_package_path
+        run: |
+          bottle_name="$(ls *.tar.gz)"
+          echo "::set-output name=bottle::$bottle_name"
+
+      - name: Get File Name
+        id: get_file_name
+        run: |
+          brew install jq
+          file_name="$(cat *.json | jq -r '."${{ env.TAP }}/${{ env.FORMULA }}".bottle.tags[].filename')"
+          echo "::set-output name=file_name::$file_name"
+
+      - name: Upload bottles as artifact
+        if: always()
```

Comment:
Can we omit the if?  Is this if TRUE?

Reply:
This is used to ensure that even on failure from previous steps, this should be executed.
But we don't have that use case here so I'll remove it.